src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -89,94 +105,54 @@ function initStore() {
                             sampleId: undefined
                         };
                     });
-            } else {
+            } else if (query.case_set_id === "all") { // case_set_id IS equal to all
```

Comment:
I feel like this comment must signify something deeper than just restating the condition, could you use that specific instead?

Reply:
good point